apps/files_sharing/src/views/SharingTab.vue

```diff
@@ -425,6 +489,30 @@ export default {
 
 	&__content {
 		padding: 0 6px;
+
+		section {
+			.section-header {
+				margin-top: 4px;
+				margin-bottom: 2px;
+				display: flex;
+				align-items: center;
+				padding-bottom: 12px;
+
+				h6 {
+					margin: 0;
+				}
+
+				:deep(.material-design-icon.information-icon) svg {
+					fill: var(--color-primary-element) !important;
+				}
```

Comment:
You could also just set `color: var(--color-primary-element)` on the icon itself (as fill is "currentColor" it will take whatever is the text color).

If using the suggestion above:

```suggestion
				.section__hint-icon {
					color: var(--color-primary-element);
				}
```